docs/src/components/Overview/OverviewTile/styles.module.css

```diff
@@ -0,0 +1,44 @@
+.overviewTile {
```

Comment:
A grid layout may be a better solution, the gaps are inconsistent now when reversed and non-reversed items are present:

![Screenshot 2024-07-30 at 09 57 15](https://github.com/user-attachments/assets/6265371e-250f-4b8d-91fd-7d55762bef79)